src/components/UserProfile/UserSettings/UserGeneralSettings/index.tsx

```diff
@@ -409,6 +419,65 @@ const UserGeneralSettings = () => {
                     </div>
                   </>
                 )}
+              {hasPermission(
+                [Permission.AUTO_REQUEST, Permission.AUTO_REQUEST_MOVIE],
+                { type: 'or' }
+              ) && (
```

Comment:
Since this feature is only usable by Plex users, should we hide these options for local users? (Similar to how we only show notification agents that have been enabled.)